### DISCUSSION TITLE
Feature Request: Allow update preference for stable, beta, alpha globally and on a per application basis.

### DISCUSSION THREAD

**Discussion by adamlogan** [Devolutions/UniGetUI#2731] [General]
I would really appreciate an option to stick to just stable releases and not have beta or alpha versions of software appear in UniGet. For example UniGetUI wants to update itself (UniGetUI) to a beta version and I'd rather not. 

My inclination is to update everything that shows up on my screen with the assumption that it is all stable. I don't want to have to look at every package individually to check if it is not stable.

I didn't see a way to to do this in the settings for UniGetUI. Let me know if I missed something.

Additionally, I would like to say I'd appreciate if this was a global option as well as a per application basis. I do use some software that is still technically Alpha, I'd like to keep updating those until they reach a stable 1.0 version and then decide whether to switch to stable going forward. Flexibility in options such as this is much appreciated, and would help a lot in making complete automation of updates attractive and practical.

**Comment by marticliment** [comment]
There are already open feature requests representing your proposals
